subsys/net/ip/net_if.c

```diff
@@ -4636,6 +4636,13 @@ bool net_if_is_offloaded(struct net_if *iface)
 		net_if_is_socket_offloaded(iface));
 }
 
+static void rejoin_multicast_groups(struct net_if *iface)
+{
+#if defined(CONFIG_NET_NATIVE_IPV6)
+	rejoin_ipv6_mcast_groups(iface);
+#endif
```

Comment:
Since you're touching this:

```suggestion
#if defined(CONFIG_NET_NATIVE_IPV6)
	rejoin_ipv6_mcast_groups(iface);
#else
	ARG_UNUSED(iface);
#endif
```